python/test/feature_assembler_test.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
```

Comment:
This statement is not required anymore (used to be with py2.6 and early 2.7s)
I added it due to `unittest` failing to see `./test/` as a module, but it didn't help...

Doesn't hurt, but modern 2.7 versions don't usually even need this...